contents/docs/libraries/react-native/index.mdx

```diff
@@ -102,7 +178,7 @@ posthog.unregister('icecream pref'),
 
 This will remove the Super Property and subsequent events will not include it.
 
-If you are doing this as part of a user logging out you can instead simply use `posthog.reset` which takes care of clearing all stored Super Properties and more.
+If you are doing this as part of a user logging out you can instead simply [`posthog.reset()`](#reset-after-logout) which takes care of clearing all stored Super Properties and more.
```

Comment:
if you use the [] to be a link, you don't need the ``

Reply:
I prefer to leave the ``, since this refers to a specific function in the code

Reply:
the problem is that if you use the ``, it's not highlighted as a link anymore, so it's a "hidden link", but no strong preferences here.

Reply:
It's still highlighted :) But not if you do it the other way round:

works: [`posthog.reset()`]
doesnt work: `[posthog.reset()]`

<img width="259" alt="Screenshot 2024-06-06 at 12 44 02 PM" src="https://github.com/PostHog/posthog.com/assets/7090054/364e5da0-ca04-453e-bd34-e277dcffe89a">